package.json

```diff
@@ -210,10 +210,11 @@
     "stylelint": "^13.7.2",
     "stylelint-scss": "^3.18.0",
     "through2": "^3.0.1",
-    "ts-jest": "^26.4.4",
+    "ts-jest": "^27.1.3",
     "ts-morph": "^9.1.0",
     "ts-node": "~8.3.0",
     "typescript": "^4.4.4",
-    "webpack-cli": "^3.3.12"
+    "webpack-cli": "^3.3.12",
+    "jest-preset-angular": "^11.1.0"
```

Comment:
Nitpick: Let's add it via yarn command: `yarn add -D jest-preset-angular` so it's placed in alphabetical order into package.json. Otherwise the future similar `yarn add` command might move your dependency alphabetically up in the list and cause unnecessary changes to the future PR.

Reply:
Oh good catch and idea, thank you!